src/components/TodoTask.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { TodoList } from '../interfaces/todo'
+
+interface Props {
+  todoList: TodoList;
+}
+
+const TodoTask = ({ todoList }: Props) => {
+  return (
+    <div className="task">
+      {todoList.map((todoItem, idx) => {
+        return (
+          <div className="content" key={idx}>
```

Comment:
Could you please also bring this out into a new component that represents a single task?

Reply:
Do you mean put the line 13 to 16 in a new component?